astroid/brain/brain_typing.py

```diff
@@ -85,6 +88,48 @@ def infer_typing_attr(node, context=None):
     return node.infer(context=context)
 
 
+GET_ITEM_TEMPLATE = """
+@classmethod
+def __getitem__(cls, value):
+    return cls
+"""
+
+
+def _looks_like_typing_alias(node: nodes.Call) -> bool:
+    """
+    Returns True if the node corresponds to a call to _alias function.
+    For example :
+
+    MutableSet = _alias(collections.abc.MutableSet, T)
+
+    :param node: call node
+    """
+    if isinstance(node, nodes.Call) and isinstance(node.func, nodes.Name):
+        if node.func.name == "_alias" and isinstance(node.args[0], nodes.Attribute):
+            return True
+    return False
```

Comment:
```suggestion
    return (
        isinstance(node, nodes.Call) and
        isinstance(node.func, nodes.Name) and
        node.func.name == "_alias" and
        isinstance(node.args[0], nodes.Attribute)
    )
```

Reply:
It might make sense to take a look at #908 after that. I updated the `pre-commit` config to use a current version of black.

Reply:
I merged #908, a rebase is probably necessary. There are a lot of pre-commit check that are in pylint and not in astroid, I might update astroid configuration once we release 2.7.2, upgraded black is enough for now.